README.md

```diff
@@ -42,6 +42,11 @@ Installation
   4. Install [Pennyworth](https://github.com/SUSE/pennyworth#installation) but instead
      of cloning the repository, install the
      [pennyworth-tool gem](https://rubygems.org/gems/pennyworth-tool/).
+     An older version is necessary now because of an API change
+     in `Pennyworth::VM#run_command`.
+
+        $ gem install pennyworth-tool --no-format-executable --version "= 0.1.0"
```

Comment:
With 0.2.0 I was getting
```
Failures:

  1) SLES 12 TFTP server  checks, tftp is running correctly
     Failure/Error: run_test_script("tftp.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b4c808>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:5:in `block (2 levels) in <top (required)>'

  2) SLES 12 TFTP server  checks, if host is in /etc/hosts
     Failure/Error: run_test_script("host.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b449a0>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:10:in `block (2 levels) in <top (required)>'

  3) SLES 12 TFTP server  checks, whether unsupported YaST modules are reported
     Failure/Error: run_test_script("unsupported_modules.sh")
     NoMethodError:
       undefined method `split' for #<Pennyworth::VM::CommandResult:0x00000002b3cc00>
     # ./helper/spec_helper.rb:30:in `run_test_script'
     # ./spec/tftp.rb:15:in `block (2 levels) in <top (required)>'

```

and indeed, [Command Result](https://github.com/SUSE/pennyworth/blob/92f43c1553c6d75fbd840d231e5cdb674e593f5a/lib/pennyworth/vm.rb#L24) does not have a split method. (0.1.0 returned a Cheetah result, which is just the captured output)